app/api.js

```diff
@@ -278,16 +278,41 @@ module.exports = function (simulation, db) {
       return res.status(400).send({ name: 'error.parsing.json', data: { message: 'Malformed event body.' }});
     }
 
-    sendEvent(instanceId, event, function (err, nextConfiguration) {
-      if (!util.IsOk(err, res)) return;
+    var queuedEvents = eventQueue[instanceId] = eventQueue[instanceId] || [];
```

Comment:
Replace with this queue logic https://github.com/JacobeanRnD/deus-ex-state-machine/blob/heroku-deploy/app/eventProcessor.js